x-pack/plugins/code/public/components/main/main.scss

```diff
@@ -271,3 +271,11 @@
 .code-no-min-width {
   min-width: unset !important;
 }
+
+.code-commit-id {
+  font-family: Menlo, $euiFontFamily;
```

Comment:
According to the EUI documentation this should be used as a mixin, like so:
`@include $euiCodeFont;`

Here's the mixin in EUI that the `@include` references:
```
@mixin euiCodeFont {
  font-family: $euiCodeFontFamily;
  letter-spacing: normal;
}
```